src/elastic/azext_elastic/aaz/latest/elastic/monitor/_update.py

```diff
@@ -104,7 +180,106 @@ def url(self):
 
         @property
         def method(self):
-            return "PATCH"
```

Comment:
3. Turning "PATCH" into "GET" & "PUT" operation in update.py causes user behavior change. Is it expected? If not, you can regenerate AAZ from codegen tool by checking operations from "GET & PUT" into "PATCH" in loading corresponding swagger file.